### ISSUE TITLE
ThreadInitThunk crashes 1 instance

### ISSUE THREAD

**Issue by Fy00zh3n** [Lolliedieb/lolMiner-releases#406]
Trying to run two instances 1 eth on rz5700xt and one for beam on vega64. Is this not possible?? Second one started for beam keeps crashing every few hours. ?? ..tried  different driver..same results (only beam 30% slower!)..

**Comment by Fy00zh3n** [comment]
Is it simply not possible to mine 2 separate algos on 2 two separate cards in one rig??

**Comment by jgonzis** [comment]
Yes you can do it... you need to run 2 instances. If it crash try a loop... or check your OC. It could be so undervolt or aggressive that makes to crash the GPU

**Comment by Fy00zh3n** [comment]
Wanted to leave this here before closing. My rig cpu is an AMD 3990x. I'm running two instances of Xmrig (XMR,QRL) and also ming Veil. Trying to maximize 64 cores as close as possible task manager indicating 86% cpu usage all cores at 3.7ghz. ETH on 5700xt was consistent 51.5mhs. Vega 64..all over + crashing (I did set up a restart loop..but it would start at 30khs and restart at 12). I added some threads to one instance of Xmrig then started ETH..it wouldn't go over 47mhs!.. reduced threads on QRL, XMR and Veil. ETH instantly wen back to 51.5mhs and Beam on Vega steady at 30khs. No crashes after several hours.

I hope this helps someone. All temps were good and I tried everything!... except reducing threads being used to mine 3 other coins!!

I figured 4 cores/8 threads left would be good enough to also run two instances of lolminer on two cards....NOT. Itried TeamRed and others and combos etc....  GREED DOESN'T PAY!  :)
